### PR TITLE
completed HTML exercises

### PR DESCRIPTION
**Pull request by Vineet-Sawant**
Please pull the latest update.

### CODE REVIEW

File: web-tech-exercises/2/clock.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+function updateClock() {
+    const clock = document.getElementById('clock');
+    const now = new Date();
```
Comment (harishkavlekar) [inline web-tech-exercises/2/clock.js:3]: Variable name can be improved

File: web-tech-exercises/2/clock.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    const hours = String(now.getHours()).padStart(2, '0');
+    const minutes = String(now.getMinutes()).padStart(2, '0');
+    const seconds = String(now.getSeconds()).padStart(2, '0');
```
Comment (harishkavlekar) [inline web-tech-exercises/2/clock.js]: try using toLocaleTimeString https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleTimeString

File: web-tech-exercises/3/script.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    company: "Infuse Consulting",
+    hobbies: ["Reading", "Listening to music", "Collecting stamps"]
+}
+
+
+
+const printUserProfile = () => {
+    // Piyush Sharma is a Senior Software Engineer at Infuse Consulting. He likes Reading, Listening to music and Collecting stamps
+    const { name, designation, company, hobbies } = user;
+    const hobbiesList = hobbies.join(', ');
```
Comment (harishkavlekar) [inline web-tech-exercises/3/script.js]: last hobby should be preceded by `and`

File: web-tech-exercises/4/style.css [hunk 4]
```diff
@@ -0,0 +1,315 @@
...
+  flex: 1;
+  padding: 10px;
+  border: 1px solid #ddd;
+}
+
+.description {
+  flex: 1;
+  padding: 10px;
+  border: 1px solid #ddd;
+}
```
Comment (harishkavlekar) [inline web-tech-exercises/4/style.css]: Styles for `.details` and `.description` can be combined

File: web-tech-exercises/5/index.html [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title></title>
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="style-movies.css">
+    </head>
+    <body>
+        <div class="search-area">
+            <h1 style="font-family: Impact, Haettenschweiler, 'Arial Narrow Bold', sans-serif; color: white"><span style="color: rgb(72, 205, 72)">M</span>ovieBaazz<span style="color: rgb(72, 205, 72)"> . . .  </span></h1>
```
Comment (harishkavlekar) [inline web-tech-exercises/5/index.html]: Styling should be moved to a separate file

File: web-tech-exercises/5/movies.js [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+function displayMovieDetails(imdbID){
+    const iframe=document.getElementById('movie-info');
+    iframe.src=`movie-details.html?imdbID=${imdbID}&apikey=${apiKey}`;
+}
+ 
+// Example usage
+// fetchMovies('Batman').then(movies => {
+//     console.log(movies); // Handle the movies data
+// });
+
```
Comment (harishkavlekar) [inline web-tech-exercises/5/movies.js]: remove commented code.

File: web-tech-exercises/5/utils.js
Comment (harishkavlekar) [inline web-tech-exercises/5/utils.js:1]: Where this file is used?

File: web-tech-exercises/5/movies.js [hunk 7]
```diff
@@ -0,0 +1,129 @@
...
+        console.error("Error fetching movies:", error);
+        return [];
+    }
+};
+ 
+const sortMoviesByYear = (movies, order) => {
+    if (order === 'asc') {
+        return movies.sort((a, b) => parseInt(a.Year) - parseInt(b.Year));
+    }
+    else if (order === 'desc') {
```
Comment (harishkavlekar) [inline web-tech-exercises/5/movies.js]: last block for `if-else` ladder should be `else`

File: web-tech-exercises/2/clock.js (hunk 1 above)
Comment (Vineet-Sawant) [reply web-tech-exercises/2/clock.js:3]: Done✅

File: web-tech-exercises/2/clock.js (hunk 2 above)
Comment (Vineet-Sawant) [reply web-tech-exercises/2/clock.js]: Done .✅

File: web-tech-exercises/4/style.css (hunk 4 above)
Comment (Vineet-Sawant) [reply web-tech-exercises/4/style.css]: done ✅

File: web-tech-exercises/5/index.html (hunk 5 above)
Comment (Vineet-Sawant) [reply web-tech-exercises/5/index.html]: done ✅

File: web-tech-exercises/5/movies.js (hunk 6 above)
Comment (Vineet-Sawant) [reply web-tech-exercises/5/movies.js]: done ✅

File: web-tech-exercises/5/utils.js
Comment (Vineet-Sawant) [reply web-tech-exercises/5/utils.js:1]: done ✅

File: web-tech-exercises/5/movies.js (hunk 7 above)
Comment (Vineet-Sawant) [reply web-tech-exercises/5/movies.js]: done ✅

File: web-tech-exercises/3/script.js (hunk 3 above)
Comment (Vineet-Sawant) [reply web-tech-exercises/3/script.js]: done ✅

**Pull request merged**